Sources/System/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2019 Apple Inc. and the Swift Numerics project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(System
+  Errno.swift
+  FileDescriptor.swift
+  FileHelpers.swift
+  FileOperations.swift
+  FilePermissions.swift
+  PlatformString.swift
+  SystemString.swift
+  Util.swift
+  UtilConsumers.swift)
+target_sources(System PRIVATE
```

Comment:
What does `PRIVATE` mean?

Reply:
In the case of `target_sources` the `PRIVATE` indicates that the listed sources are not meant to be used to populate `INTERFACE_SOURCES` for the target.  This means that the sources listed here do not get propagated to users of this library.

Reply:
But do top-level sources? Unlike the `Internals` folder, this is just a way of organizing `FilePath` code together for navigation purposes. There's still API inside.

Reply:
ah, I see your point.  No, the top-level sources are treated as private by default.  I only did the split `target_sources` for reading.  We can even merge them into a single call if you prefer.  That is a stylistic thing.

Reply:
I have no opinions here, just hoping to understand CMake a little bit better :-)